specification/common-types/resource-management/v6/types.json

```diff
@@ -287,12 +337,48 @@
         "error": {
           "$ref": "#/definitions/ErrorDetail",
           "description": "If present, details of the operation error."
+        },
+        "resourceId": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "Fully qualified ID of the resource against which the original async operation was started.",
+          "readOnly": true
         }
       },
       "required": [
         "status"
       ]
     },
+    "Origin": {
+      "type": "string",
+      "description": "The intended executor of the operation; as in Resource Based Access Control (RBAC) and audit logs UX. Default value is \"user,system\"",
+      "enum": [
+        "user",
+        "system",
+        "user,system"
+      ],
+      "x-ms-enum": {
+        "name": "Origin",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "user",
+            "value": "user",
```

Comment:
>"user" [](http://example.com/codeflow?start=21&length=6)

Shouldn't these be capitalized? This is the most common convention, e.g. User, System. The other common types use capitalized enum values as well.